src/vm/eventpipefile.cpp

```diff
@@ -190,6 +217,7 @@ void EventPipeFile::WriteEvent(EventPipeEventInstance &instance, ULONGLONG captu
     CONTRACTL_END;
 
 #ifdef DEBUG
+    ASSERT(m_isInitialized);
```

Comment:
missing #endif?

Reply:
nvm. read that wrong :p

Reply:
The code below was already inside an `#ifdef DEBUG` so I just put them together